ci-operator/step-registry/redhat-chaos/pod-scenarios/random-system-pods/redhat-chaos-pod-scenarios-random-system-pods-commands.sh

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
do you want to do the same thing for this sceanrio as [here](https://github.com/openshift/release/blob/0de1950ff6de772f7d452dfde87ba9da893e0bd7/ci-operator/step-registry/redhat-chaos/pod-scenarios/etcd-disruption/redhat-chaos-pod-scenarios-etcd-disruption-commands.sh)

Reply:
Done!